doc/source/ray-air/examples/lightgbm_example.ipynb

```diff
@@ -142,6 +141,7 @@
     "        datasets={\"train\": train_dataset, \"valid\": valid_dataset},\n",
     "        preprocessor=preprocessor,\n",
     "        num_boost_round=100,\n",
+    "        metadata = {\"preprocessor_pkl\": preprocessor.pickle()}\n",
```

Comment:
this doesn't work right now--- you need to apply the preprocessor beforehand rather than passing to the Trainer so you will pickle the fitted state.

Reply:
Oh woops thought I moved it out - thanks for catching